packages/components/tests/dummy/app/styles/app.scss

```diff
@@ -23,7 +23,6 @@
 @import "./showcase-components/outliner";
 @import "./showcase-components/placeholder";
 
-
```

Comment:
the two lines were on purpose :|

Reply:
I believe this occurred at the same time as my changing the the double quotes to single quotes, which I think was prettier being a bit to heavy handed. I've been having issues with it especially related to markdown, let me take a look at my config.